_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+It updates in real time, with a little counter and a red bar. The audacity.
 <img class="no-flow vertical" src="/img/screenshots/progress_bar.png"/>
 
-All of what you've read above was made possible by the lean and mean coding machine, the one and only - (drumroll) - [@avently](https://github.com/avently)! Seriously, you should all sacrifice your first born child in his name. Or just say thank you, whatever works out for ya (that would be kind of boring, though).
+This fine list of changes were made possible by the steadily awesome [@avently](https://github.com/avently). Let him live rent free in your head, as you bask in the fruits of his labour.
 
 * * *
 
-On a serious note, this was a massive, massive change to not only the code, but also the app's workflow, so this PR was tested, reviewed and modified lots of times, over a long period of time, before it was allowed to see the light of day with this release. A huge thank you to you, [@avently](https://github.com/avently), for having the patience and dedication to keep adding features and bug fixes until everyone was satisfied! A different person probably would have gone insane. And a big thank you to the maintainers for doggedly reviewing this gigantic PR repeatedly. Those ~6600 additions and ~5600 removals must have felt intimidating!
+This massive, massive change to not only the code, but also the app's workflow, meant the PR had to be tested, reviewed and modified lots of times, over a long period of time. A huge thank you to [@avently](https://github.com/avently), for having the patience and dedication to keep adding features and bug-fixes until everyone was satisfied! A different person probably would have gone insane. And a big thank you to the maintainers for doggedly reviewing this gigantic PR repeatedly. Totalling ~6600 additions and ~5600 removals, the hat is off.
 
-Naturally, since this was such a massive change, several bugs also slipped through. But [@avently](https://github.com/avently), with the help of [@blackbox87](https://github.com/blackbox87), [@Stypox](https://github.com/Stypox) and [@TobiGr](https://github.com/TobiGr) fixed them rapidly as they were reported. 
+This evolution saw evolutionary bugs in its wake. [@avently](https://github.com/avently), with the help of [@blackbox87](https://github.com/blackbox87), [@Stypox](https://github.com/Stypox) and [@TobiGr](https://github.com/TobiGr) swatted as the reports came in. 
 
 ## The Notification is dead, long live the Notification!
 
-If you care about such things, you have likely noticed that NewPipe's notification looks quite different from most media player apps these days. It has a progress bar, buttons in odd places, and a permanent grey colour. This is because NewPipe uses an old API for notifications called RemoteView. This allowed for a lot of customisation, and that was good at the time it was in vogue. However, it has been deprecated and superseded by a more modern API.
```

Comment:
What are "such things". The above, the following.??? OK, the following. What? Why is that different. No, I haven't noticed. Ah, is it? Well. Why? Why does the reader care it is different, and what its technical name is. OK, the reader now knows all that, and for what? "deprecated and superseded". OK, discarding. Modern API. OK, there is a more modern API now for notifications. That is all the content worth taking in this whole section.

Reply:
> No, I haven't noticed.

That means you _don't_ care about such things. ;) What's the problem? Just skip this section, as other readers will.

> Why does the reader care it is different, and what its technical name is.

Does giving the name somehow cause the end of the world? It is just trivia for those who are interested. Those who aren't can just skip it or forget about it.

> That is all the content worth taking in this whole section.

Again, for those who "don't care" like the very first sentence says, in which case, just move on.

Reply:
So what could be interesting isn't, because it doesn't matter when it isn't? Solid logic.